uobtheatre/bookings/models.py

```diff
@@ -681,7 +683,14 @@ class Ticket(models.Model):
     )
     seat = models.ForeignKey(Seat, on_delete=models.RESTRICT, null=True, blank=True)
 
-    checked_in = models.BooleanField(default=False)
+    checked_in_at = models.DateTimeField(null=True, blank=True, default=None)
+    checked_in_by = models.ForeignKey(
+        User,
+        on_delete=models.RESTRICT,
+        related_name="tickets_checked_in_by_me",
```

Comment:
```suggestion
        related_name="ticked_checked_in_by_user",
```
Then delete and regenerate your migrations